Sources/KeyboardSupport/UIView+KeyboardSupport.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  UIView+KeyboardSupport.swift
+//  KeyboardSupport
+//
+//  Created by John Davis on 12/3/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    
+    /// highest level superview of the view hierarchy
+    var topLevelContainer: UIView {
+        var returnView = self
+        
+        while returnView.superview != nil {
+            returnView = returnView.superview!
+        }
+        
+        return returnView
+    }
+    
+    /// Walks the view hierachy starting with itself to build an array of non-hidden,
+    /// UITextInput subviews that can become first responder.
+    ///
+    /// - Returns: Array of UITextInput subviews
+    func getAllTextInputViews() -> [UITextInputView] {
+        var fields: [UITextInputView] = []
+        
+        subviews.forEach { subview in
+            guard !subview.isHidden else { return }
```

Comment:
I think we'd want `continue` instead of `return`, right? `return` will break us out of the loop while `continue` will simply skip that subview.

Reply:
I tried that first, but because this is actually a block being executed with each element in the array, return is the right way to go. 

Trying to use "continue" complains that you're not using it in a loop, lol.